kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/ExecWebSocketListener.java

```diff
@@ -95,14 +97,13 @@ public void close() {
             if (!executorService.awaitTermination(10, TimeUnit.SECONDS)) {
                 executorService.shutdownNow();
             }
-        } catch (Throwable t) {
-            throw KubernetesClientException.launderThrowable(t);
-        }
-
-        WebSocket ws = webSocketRef.get();
-        try {
-            if (ws != null) {
-                ws.close(1000, "Closing...");
+            if (client != null) {
```

Comment:
i just want send a command to  apiserver to get result, i don't want keep watching, i need to shutdown all the running thread.

ws.close() is not equivalent to calling client.close(), client.close will shutdown okhttp thread.
If just ws.close(), okhttps threads are still alive.Of course the okhttp thread will close automaticlly after a minutes if there is no request.